lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/11_crosscutting_group/adjudication_amount.rb

```diff
@@ -0,0 +1,46 @@
+module CarinForBlueButtonTestKit
+  class AdjudicationAmountTest < Inferno::Test
+    title 'Properly uses ".adjudication.amount"'
+
+    description <<~DESCRIPTION
+      For:
+        -  C4BB ExplanationOfBenefit Inpatient Institutional `.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Outpatient Institutional `.adjudication.amount`
+
+      The Health IT Module must only populate `.adjudication.amount` if `.item.adjudication` is not available.
+    DESCRIPTION
+
+    id :adjudication_amount
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@143',
```

Comment:
143, 154, 155 are all out of scope

Reply:
modified to be in scope. let Karl know. Will regenerate files.

Reply:
the CARIN reqs were one of the first extracted, and it looks like the running definition (or at least an initial interpretation) was "can we automate this?". But several of these can actually be attested for. I'm going to review, update, and regenerate the requirements.